.travis.yml

```diff
@@ -30,7 +30,7 @@ install:
   - "time (bash -e .travis/misc-setup.sh | ts)"
   - "cp .travis/localsettings.py localsettings.py"
   - "pip install coverage unittest2 mock --use-mirrors"
-script: "coverage run manage.py test --noinput --failfast --traceback --testrunner=$TEST_RUNNER"
+script: "coverage run manage.py test --noinput --traceback --testrunner=$TEST_RUNNER"
```

Comment:
i think so. i think it'll add a lot of time to the tests. double checking with @dannyroberts